tasks/taxon_id/task_nextclade.wdl

```diff
@@ -1,175 +1,175 @@
-      tamiflu_aa_subs = tamiflu_aa_subs + "~{tamiflu_aa_substitutions}".split(',')
-
-      def intersection(lst1, lst2):
-        # returns intersection between nextclade identified aa substitutions and
-        # tamiflu associated aa substitutions
-        return list(set(lst1) & set(lst2))
-
-      with codecs.open("./input.tsv",'r') as tsv_file:
-        tsv_reader = csv.reader(tsv_file, delimiter="\t")
-        tsv_data = list(tsv_reader)
-
-        if len(tsv_data) == 1:
-          tsv_data.append(['NA']*len(tsv_data[0]))
-        tsv_dict = dict(zip(tsv_data[0], tsv_data[1]))
-
-        # combine 'clade_nextstrain' and 'clade_who' column if sars-cov-2, if false then parse 'clade' column
-        if ("~{organism}" == "sars-cov-2"):
-          with codecs.open("NEXTCLADE_CLADE", 'wt') as Nextclade_Clade:
-            nc_clade = tsv_dict['clade_nextstrain']
-            who_clade = tsv_dict['clade_who']
-            if (nc_clade != who_clade) and (nc_clade != '') and (who_clade != ''):
-              nc_clade = nc_clade + " (" + who_clade + ")"
-            if nc_clade == '':
-              nc_clade = 'NA'
-            Nextclade_Clade.write(nc_clade)
+  meta {
+    description: "Python and bash codeblocks for parsing the output files from Nextclade."
+  }
+  input {
+    File nextclade_tsv
+    String docker = "python:slim"
```

Comment:
FYI/warning, we may want to consider switching to a static docker image. The `python:slim` docker image from dockerhub is updated frequently without warning: https://hub.docker.com/_/python/tags?page=1&name=slim

I haven't seen this fail ever, as it's some pretty vanilla python code but would be good to consider